src/Microsoft.AspNet.Mvc.Core/ModelBinding/MutableObjectModelBinder.cs

```diff
@@ -98,6 +98,12 @@ protected virtual bool CanUpdateProperty([NotNull] ModelMetadata propertyMetadat
                 return true;
             }
 
+            // Create the object if it is top level object and there are no properties to bind
```

Comment:
Could we change the comment to follow the numbering scheme that the previous comment starts?